libs/distancing.py

```diff
@@ -68,6 +68,9 @@ def __init__(self, config, source, live_feed_enabled=True):
         if not os.path.exists(self.screenshot_path):
             os.makedirs(self.screenshot_path)
 
+        if "Classifier" in self.config.get_sections():
+            self.face_threshold = float(self.config.get_section_dict("Classifier").get("MinScore", 0.75))
```

Comment:
Is there a special reason to use 0.75 as default score threshold, and not 0.5 for example?

Reply:
I put the value 0.75 to improve the accuracy in the mask detection reducing the number of false positives.